Source/Core/Core/IOS/MIOS.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+
+namespace IOS
+{
+namespace HLE
+{
+namespace MIOS
+{
+bool Load();
+
+// Source: https://wiibrew.org/wiki/ARM_Binaries
+struct ARMBinaryHeader
```

Comment:
Structs, enums, etc, should preferably be defined before functions. If this is only used internally, it should likely be placed within the cpp file.